conan/cli/printers/graph.py

```diff
@@ -39,12 +39,17 @@ def _format_requires(title, reqs_to_print):
         if not reqs_to_print:
             return
         output.info(title, Color.BRIGHT_YELLOW)
-        for ref, (recipe, remote, test_package) in sorted(reqs_to_print.items()):
+        for ref, (status, remote, test_package) in sorted(reqs_to_print.items()):
```

Comment:
Recipes should be left as-is, it is only the binaries that should be skipped

Reply:
I don't get what you mean. Here there are no changes at all, I've only renamed the `recipe` variable to `status` because it's actually the status and not the recipe itself